Assignment2/FORGIS98_JorgeSol.ttl

```diff
@@ -0,0 +1,12 @@
+@base <http://example.org/> .
+@prefix data: <http://example.org/data> .
+
+:Class01 :includes ( :Sensor029 :Computer101 ) .
```

Comment:
This is not right, indeed. What you would do is
:Class01 :includes :Sensro029 , :Computer101 .

Reply:
Perfect, does the comma need space around it? Or it doesn't matter?

Reply:
It does not matter